chat_shell/chat_shell/tools/sandbox/_base.py

```
@@ -0,0 +1,497 @@
+
+        # Ensure E2B SDK is patched
+        patch_e2b_sdk()
+
+    @classmethod
+    def get_instance(
+        cls,
+        task_id: int,
+        user_id: int,
+        user_name: str,
+        timeout: int = DEFAULT_SANDBOX_TIMEOUT,
+        bot_config: list = None,
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)